pkg/trace/api/profiles_test.go

```diff
@@ -47,7 +61,8 @@ func TestProfileProxy(t *testing.T) {
 		t.Fatal(err)
 	}
 	rec := httptest.NewRecorder()
-	newProfileProxy(nil, u, "123", "key:val").ServeHTTP(rec, req)
+	c := &traceconfig.AgentConfig{}
```

Comment:
Is this used anywhere? It seems not.

Reply:
It's used below to create the transport - I used the config because I couldn't find another more convenient constructor.

I'm not sure why the test previously passed nil as the transport, but if we do this now the test fails.

Reply:
Ah, yes, doooh. :)